package/MDAnalysis/topology/MOL2Parser.py

```diff
@@ -148,9 +180,37 @@ def parse(self, **kwargs):
         resids = []
         resnames = []
         charges = []
-
+        has_charges = False if sections['molecule'][3].strip() == 'NO_CHARGES'\
+            else True
```

Comment:
What I was thinking here was
```suggestion
        has_charges = sections['molecule'][3].strip() != 'NO_CHARGES'
```

When you have `True if CONDITION else False` you can work directly with `CONDITION` since it already returns a `bool`.